lib/Graph/Grad.cpp

```diff
@@ -319,9 +319,9 @@ Function *glow::differentiate(Function *F, const TrainingConfig &conf,
       // to the correspondent Data Tensors, where Vector value
       // points to Data Tensor.
       auto *SN = G->createSplat("splat.grad", Data.getType(), 0);
-      auto *SA = new ScatterAssignNode("scatter.assign.grad", SN->getResult(),
-                                       RI->getResult(),
-                                       RG ? RG->getResult() : OutputG);
+      auto *SA = new ScatterDataNode(
+          "scatter.assign.grad", SN->getResult(), RI->getResult(),
+          RG ? RG->getResult() : OutputG, /*Add*/ false);
```

Comment:
tiny nit: In most other places you're calling it `/*Cumulative*/`. Don't care either way, just prefer to be consistent.

Reply:
Thanks for pointing out! Yeah, I renamed this during development, and it seems that vestige of old name is still in code. I'll review it carefully.